android/KMEA/app/src/main/java/com/tavultesoft/kmea/util/HelpFile.java

```diff
@@ -47,8 +49,11 @@ public static Intent toActionView(Context context, String helpFile, String packa
         i.addFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
         i.addFlags(Intent.FLAG_ACTIVITY_NO_HISTORY);
       } catch (NullPointerException e) {
-        String message = "FileProvider undefined in app to load" + customHelp.toString();
-        Toast.makeText(context, message, Toast.LENGTH_LONG).show();
+        // Localize Toast error
+        BaseActivity.makeToast(context, R.string.fileprovider_undefined, Toast.LENGTH_LONG);
```

Comment:
I chose not to use the formatted string for the Toast here, since the full path to the help file will just be gibberish to the user (and quite long)